pinot-query-runtime/src/test/resources/queries/BasicQuery.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "basic_test": {
+    "description": "basic test case example",
+    "sql": "SELECT * FROM {tbl}",
+    "tables": {
+      "tbl": { "col1": "STRING", "col2": "INT" }
```

Comment:
since order matters in the `inputs` part of the model, we should model this part as an array of tuples (as suggested in comment above) instead of a map since JSON can reorder maps if it wants:
```
"tbl": [{"name":"col1", "type":"STRING}, {"name":"col2", "type":"INT"}]
```

Let's commit this first, I think I might be able to write a simple utility that allows us to declare tables using SQL syntax and then turn it into what we want:
```
"tbl": "col1 STRING, col2 INT"
```

Reply:
you are right. i was thinking about ordered map in yaml. let me rewrite this

Reply:
done